src/psycop_feature_generation/loaders/raw/load_visits.py

```diff
@@ -84,27 +104,27 @@ def physical_visits(
             "emergency_visits": "'Akut ambulant'",
         }
         d = {key: d[key] for key in ["LPR3", visit_type]}
-        d["LPR3"]["where"] += f" AND pt_type = {LPR3_types[visit_type]}"
+        d["LPR3"].where += f" AND pt_type = {LPR3_types[visit_type]}"
 
     dfs = []
 
     for meta in d.values():
```

Comment:
```suggestion
    for schema in d.values():
```

Rename in the below as well. Makes it much easier to read :-)